demo/README.md

```diff
@@ -0,0 +1,60 @@
+
+### Goal: Bring up Backend client for Fabric Auction demo network
+
+Some dependencies are currently hardcoded. This is work in progress.  As the scripts to stand up the demo application evolve, these will be modified. Example: NETWORK_NAME used in following instructions.
+
+Usage:
+```
+## Build image once
+cd fabric-private-chaincode/demo/client/backend
+docker build -t auction_backend  .
+```
+
```

Comment:
you might add a short section that you assume a network setup via $FPC_PATH/utils/docker-compose with `USE_FPC=false`.

Reply:
> you might add a short section that you assume a network setup via $FPC_PATH/utils/docker-compose with `USE_FPC=false`.

why specifically 'USE_FPC=false' ?  For the client, it does not matter if it is FPC or Fabric network

Reply:
> > you might add a short section that you assume a network setup via $FPC_PATH/utils/docker-compose with `USE_FPC=false`.
> 
> why specifically 'USE_FPC=false' ? For the client, it does not matter if it is FPC or Fabric network.  

Added a sentence which says "Before the following steps can be run, an FPC network should be setup using..."

Reply:
your subsequent instruction would work only for USE_FPC=false as (a) the network-name is different for USE_FPC=true and (b) you have to invoke installs directly in peer using ${PEER_CMD} rather than doing it in cli docker ...

Reply:
1.  removing the scripts installCC,.sh and upgradeCC.sh from this PR;   reference to these scripts will be removed from README. 

2.  can we have the same network name for both cases of USE_FPC=true or false?
  
For this PR, I will change the note that (this is work-in-progress;) the NETWORK_NAME is dependent on USE_FPC as mentioned in README.  Will instruct the user to verify the network name and set it accordingly.

Reply:
BTW, when I was integrating FPC peer,  Bruno's chaincode and the backend client, I had modified installCC.sh to install / upgrade CC on FPC network - by running {PEER_CMD} instead of  fabric "peer" inside peer container - _from cli / script_.  I have shared the modified scripts / notes  with Swetha.